bin/load-initial-html.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+
+if [[ "${DEVICE_TESTS}" = true ]] ; then
+    cp ./bin/tmp/initial-device-tests-html.js ./src/app/initial-html.js
```

Comment:
You probably have tried it in the past but, will ask just to be sure: instead of copying/overwriting files, have you perhaps tried using the `process.env.DEVICE_TESTS` environment variable directly in `src/app/initial-html.js` to export the desired html content?

Reply:
Yes I have, the environment variable won't be available when the app is being bundled. I've also tried using [babel-plugin-transform-inline-environment-variables](https://www.npmjs.com/package/babel-plugin-transform-inline-environment-variables) but this also doesn't work during bundling for the release version.

Reply:
Interesting. I think I'd like to dig into that to understand how come the bundling doesn't pick up env variables. Nothing to block this PR though 👍

Reply:
Yeah it works when specifying the environment variables on `yarn start` so for example `VAR=foo yarn start` but otherwise no luck, and I basically spent a day trying to find a way to accomplish this but I didn't want to spend too much time on it right now. I did plan to circle back to it as well too so for sure let me know if you're able to come up with something here.